### PR TITLE
README Update

### PR DESCRIPTION
**Pull request by jordigarcl**
This PR updates the README to better clarify current users how to download and use the extension. It also provides updated context on the fork state and updates the License so Github recognizes it as zlib.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,8 +1,4 @@
-# LICENSE
-
-zlib License
-
-Copyright (C) 2020 Animal Crossing Music Extension
+Copyright (C) 2020 PikaDude
```
Comment (jordigarcl) [inline LICENSE]: I didn't know what to put here so I just put @PikaDude 's username as this is living in his repo 🤷‍♂️

File: README.md [hunk 2]
```diff
@@ -16,64 +15,71 @@ Listen to the corresponding hourly music from the main Animal Crossing titles:
...
 
+With it you can compone your custom Town Tune, which will be played every time an hour passes (just as in the game!)
 
 ## Installation
-The changes in this repository **remain unpublished** in any way. 
+Notice that this project remains unpublished right now. You'll soon be able to install it through the Chrome Web Store!
 
-Ideally, we would like to update the [original extension](https://chrome.google.com/webstore/detail/animal-crossing-music/fcedlaimpcfgpnfdgjbmmfibkklpioop) so project development could continue with the same user-base. Otherwise, this project will be published as a new extension.
+### Source Code
+You can [download](https://github.com/PikaDude/Animal-Crossing-Music-Extension/archive/develop.zip) the source code of the project and install the unpacked extension yourself.
```
Comment (jordigarcl) [inline README.md]: Even though `develop` was the base branch I suspect users were clicking this link and therefore downloading the broken `master`. Notice I changed it to download `develop`. We should revert this once we make `master` stable again.

File: README.md [hunk 3]
```diff
@@ -16,64 +15,71 @@ Listen to the corresponding hourly music from the main Animal Crossing titles:
...
 
-### Local Installation
-In the meantime, you can [download](https://github.com/PikaDude/Animal-Crossing-Music-Extension/archive/master.zip) the latest stable version of the project and install it locally. 
+To install the downloaded project, do the following:
+1. Unzip the downloaded file
```
Comment (jordigarcl) [inline README.md:41]: Added an unzip step just in case

File: README.md [hunk 4]
```diff
@@ -16,64 +15,71 @@ Listen to the corresponding hourly music from the main Animal Crossing titles:
...
+### Issues
+If you want to report a bug, or have a feature request/proposal for improvement, feel free to open a public [issue](https://github.com/PikaDude/Animal-Crossing-Music-Extension/issues) on the project repository.
 
-[PRs](https://github.com/PikaDude/Animal-Crossing-Music-Extension/pulls) are equally welcomed. Just make sure to mention a relevant issue in its description!
+### PR Submission
+[PRs](https://github.com/PikaDude/Animal-Crossing-Music-Extension/pulls) are equally welcomed. Just make sure to:
 
+* Any changes you introduce **do not** break the project in any way
+* You have mentioned any relevant issue in its description
+* You choose `develop` as base branch
```
Comment (jordigarcl) [inline README.md]: I came up with these PR requirements on the spot, do you agree?

File: README.md (hunk 2 above)
Comment (PikaDude) [reply README.md]: I think we should just tell people to press the download ZIP button at the top as it's more automatic and relative if you know what I mean.

Comment (jordigarcl) [reply README.md]: I have also added a download button near the top of the README 👍

Comment (PikaDude) [reply README.md]: chrome_2020-01-17_16-05-36
I was kinda referring to the GitHub download button lol

Comment (jordigarcl) [reply README.md]: Oh I know but I'd expect that people either 1) are not familiar with Github and need instructions, therefore '**clone** or download' may be tricky for them, or 2) already knows how Github works and wouldn't need instructions in the first place

Comment (PikaDude) [reply README.md]: True true

Comment (jordigarcl) [reply README.md]: That's just my opinion though, do you think we should still remove the download badge and/or tell them to click the github button?

Comment (PikaDude) [reply README.md]: Yeah otherwise it might get confusing

File: LICENSE (hunk 1 above)
Comment (PikaDude) [reply LICENSE]: I'm not sure if it would be valid to do this.

Comment (jordigarcl) [reply LICENSE]: Actually I did some research at the beginning of the fork and it looks like best way to procede is to put current copyright holders first then include the original line as 'partial copyright'

File: README.md (hunk 4 above)
Comment (PikaDude) [reply README.md]: Trying to ask people not to submit PRs that break the project is kinda vague and sometimes hard for them to confirm and verify, it's best that we do these tests ourselves and don't bother mentioning it.

Comment (jordigarcl) [reply README.md]: Then I think we should just point out that they should submit them`develop`

File: LICENSE (hunk 1 above)
Comment (jordigarcl) [reply LICENSE]: I'll discard this change seeing how we will probably publish it under the ACME gh org (wow I like that acronym!)

**Comment by jordigarcl** [comment]
Rebased against develop and made some changes

**Review by PikaDude — APPROVED** [review]

**Pull request merged**